app/plugins/ee-openid-connect.md

```diff
@@ -0,0 +1,130 @@
+---
+id: page-plugin
+title: Plugins - OpenID Connect
+header_title: OpenID Connect
+header_icon: /assets/images/icons/plugins/ee-openid-connect.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+OpenID Connect ([1.0][connect]) plugin allows the integration with a 3rd party
+identity provider (IdP) or [Kong OAuth 2.0 Plugin][oauth2plugin] in a standardized way.
+This plugin can be used to implement Kong as a (proxying) [OAuth 2.0][oauth2] resource
+server (RS) and / or as a OpenID Connect relying party (RP) between the client
+and the upstream service.
+
+The plugin supports several types of credentials. Currently it can be used with
+the following types of credentials:
+
+- Signed [JWT][jwt] access tokens ([JWS][jws]) with the standardised signing algorithms ([JWA][jwa])
+- Opaque access tokens with either Kong OAuth 2.0 plugin issued tokes or 
+  3rd party IdP issued ones through token introspection (IdP needs to support it)
+- Username and password through the OAuth 2.0  password grant (the plugin will
+  automatically exchange such credentials with access token by calling IdP's token
+  endpoint)
+- Client id and secret through the OAuth 2.0 client credentials grant (the plugin
+  will automatically exchange such credentials with access token by calling IdP's
+  token endpoint)
+- Authorization code that plugin can retrieve from the client when using
```

Comment:
Which "plugin" are we referring to here? "Plugins can", as in other plugins, in a general sense, or one from this OIDC suite in particular? Not sure